bin/xunion

```diff
@@ -4,14 +4,31 @@ const XUnion = require('../lib/xunion');
 const commandLineArgs = require('command-line-args');
 
 const argDefinitions = [
-  { name: 'peerport', type: Number },
-  { name: 'rpcport', type: Number },
+  { name: 'p2p.port', type: Number },
+  { name: 'rpcPort', type: Number },
   { name: 'database', type: String },
   { name: 'xudir', type: String },
   { name: 'lnddir', type: String },
 ];
 const args = commandLineArgs(argDefinitions);
 
-const xUnion = new XUnion(args);
+const argKeys = Object.keys(args);
```

Comment:
Rather export this processing to a generic util function.
Benefits:
- reusable
- testable
- easier to understand what it's doing

Reply:
@moshababo Thanks, done.